yt_dlp/extractor/digitalconcerthall.py

```diff
@@ -119,31 +131,31 @@ def _entries(self, items, language, type_, **kwargs):
             }
 
     def _real_extract(self, url):
-        language, type_, video_id = self._match_valid_url(url).group('language', 'type', 'id')
+        language, type_, video_id, part = self._match_valid_url(url).group('language', 'type', 'id', 'part')
         if not language:
             language = 'en'
 
-        thumbnail_url = self._html_search_regex(
-            r'(https?://images\.digitalconcerthall\.com/cms/thumbnails/.*\.jpg)',
-            self._download_webpage(url, video_id), 'thumbnail')
-        thumbnails = [{
-            'url': thumbnail_url,
-            **parse_resolution(thumbnail_url),
-        }]
-
+        api_type = 'concert' if type_ == 'work' else type_
         vid_info = self._download_json(
-            f'https://api.digitalconcerthall.com/v2/{type_}/{video_id}', video_id, headers={
+            f'https://api.digitalconcerthall.com/v2/{api_type}/{video_id}', video_id, headers={
                 'Accept': 'application/json',
                 'Accept-Language': language,
             })
-        album_artist = ' / '.join(traverse_obj(vid_info, ('_links', 'artist', ..., 'name')) or '')
+        album_artists = traverse_obj(vid_info, ('_links', 'artist', ..., 'name'))
         videos = [vid_info] if type_ == 'film' else traverse_obj(vid_info, ('_embedded', ..., ...))
 
+        if type_ == 'work':
+            videos = [videos[int(part) - 1]]
+
+        thumbnail = traverse_obj(vid_info, (
+            'image', ..., {lambda x: f'https:{x}' if x.startswith('//') else x},
+            {lambda x: x.replace('{width}x{height}', '0x0')}, {url_or_none}, any))  # NB: 0x0 is the original size
```

Comment:
use `self._proto_relative_url`

```suggestion
            'image', ..., {self._proto_relative_url}, {url_or_none},
            {lambda x: x.replace('{width}x{height}', '0x0')}, any))  # NB: 0x0 is the original size
```

also could do this, but it might not be as obvious what we're doing 

```suggestion
            'image', ..., {self._proto_relative_url}, {url_or_none},
            {lambda x: x.format(width=0, height=0)}, any))  # NB: 0x0 is the original size
```